src/modules/headset/headset_openxr.c

```diff
@@ -330,6 +330,11 @@ static bool openxr_init(HeadsetConfig* config) {
 #ifdef LOVR_VK
       { "XR_KHR_vulkan_enable2", NULL, true },
 #endif
+
+#ifdef __ANDROID__
+      { "XR_KHR_android_create_instance", NULL, true },
+      { "XR_PICO_android_controller_function_ext_enable", NULL, true},
```

Comment:
Because the 'feature' here is `NULL`, the extension will always get added.  Does this mean the instance will fail to create on other OpenXR runtimes that don't support the extension?

Reply:
`XR_PICO_android_controller_function_ext_enable` needs to be removed, because it's unused. `XR_KHR_android_create_instance` are added for android, and Android OpenXR environments (Pico, Oculus, Monado-Android) supports it.